site/docs/aliyun.md

```diff
@@ -0,0 +1,145 @@
+
+## Enabling Aliyun Integration
+
+The `iceberg-aliyun` module integrates alibaba cloud services (Aliyun OSS, Aliyun DLF etc) with apache iceberg.
+Currently, it provides the bundled `iceberg-aliyun-runtime` module for users to access the iceberg table backed in
+alibaba cloud services. To enable the aliyun integration, people only need to ensure that the `iceberg-aliyun-runtime`
+jar is loaded into classpath correctly by the engines such as Spark, Flink, Hive, Presto etc.
+
+## Catalogs
+
+[Aliyun DLF](https://www.aliyun.com/product/bigdata/dlf) is a core service from Alibaba Cloud that satisfies users'
+needs for data asset management while creating data lake tables. DLF provides unified metadata views and permission
+management for data available in OSS ([Aliyun Object Storage Service](https://www.alibabacloud.com/product/object-storage-service)).
+It also provides real-time lake migration and cleaning templates for data and production-level metadata services for
+upper-layer data analysis engines. Aliyun DLF is a good choice to manage the apache iceberg tables, the Aliyun DLF
+catalog integration will come in the next releases.
+
+### Engines Access.
+
+All the engines (Spark, Hive, Flink, Presto) can access the iceberg table backed in aliyun services. There are following
+examples to show how to access it.
+
+The ideal way to show the example is using Aliyun DLF Catalog to manage those iceberg tables, but we still don't finish
+the iceberg + DLF integration work in apache iceberg repository. Here we are showing the examples to manage iceberg
+tables in Hive Catalog.
+
+### Spark
+
+For example, to access apache iceberg tables stored in alibaba object storage service with Apache Spark 3.2.x:
+
+```bash
+# Add Iceberg dependency
+ICEBERG_VERSION=0.13.0
+ALIYUN_ACCESS_KEY_ID=******      # Your Aliyun access key id.
+ALIYUN_ACCESS_KEY_SECRET=******  # Your Aliyun access key secret.
+ALIYUN_OSS_ENDPOINT=******       # Your Aliyun OSS endpoint.
+
+DEPENDENCIES="org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:$ICEBERG_VERSION"
+DEPENDENCIES+=",org.apache.iceberg:iceberg-aliyun-runtime:$ICEBERG_VERSION"
+
+# Start Spark SQL client shell
+spark-sql --packages $DEPENDENCIES \
+    --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions \
+    --conf spark.sql.catalog.my_catalog=org.apache.iceberg.spark.SparkCatalog \
+    --conf spark.sql.catalog.my_catalog.uri=thrift://<host>:<port> \
+    --conf spark.sql.catalog.my_catalog.warehouse=oss://my-bucket/my/key/prefix \
+    --conf spark.sql.catalog.my_catalog.io-impl=org.apache.iceberg.aliyun.oss.OSSFileIO \
+    --conf spark.sql.catalog.my_catalog.oss.endpoint=$ALIYUN_OSS_ENDPOINT \
+    --conf spark.sql.catalog.my_catalog.client.access-key-id=$ALIYUN_ACCESS_KEY_ID \
+    --conf spark.sql.catalog.my_catalog.client.access-key-secret=$ALIYUN_ACCESS_KEY_SECRET
+```
+
+Let's create iceberg tables and insert few records into it.
```

Comment:
nit: since the example is just creating one table this should be "Let's create an iceberg table and insert a few records into it using Spark-SQL."